package.json

```diff
@@ -51,8 +51,8 @@
     "typescript": "^5.0.2"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": ">=4.0.0",
-    "@typescript-eslint/parser": ">=4.0.0",
+    "@typescript-eslint/eslint-plugin": "^7.0.0",
+    "@typescript-eslint/parser": "^7.0.0",
     "eslint": ">=8.0.0",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏We should pin ESLint itself to v8 so the peer deps warn someone trying to use v9

```suggestion
    "eslint": "^8.0.0",
```

Reply:
Done